generators/init/constants.mjs

```diff
@@ -16,7 +16,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-export const NODE_VERSION = '14.18.1';
+export const NODE_VERSION = '18.16.1';
```

Comment:
@mshima We should use only 1 constant for NODE_VERSION, no?
Maybe use only the one from init generator?

Reply:
NODE_VERSION at init is used as required node version:
https://github.com/jhipster/generator-jhipster/blob/61ec0be217e40fa96c944bdc624921efb5104bb6/generators/init/templates/package.json.ejs#L35

I think it should match https://github.com/jhipster/generator-jhipster/blob/61ec0be217e40fa96c944bdc624921efb5104bb6/package.json#L179.
We probably should copy the package.json version.